### DISCUSSION TITLE
Apply windows 11 automatic accent color in more places.

### DISCUSSION THREAD

**Discussion by duckduckdog** [ramensoftware/windhawk-mods#251] [Mod ideas]
Windows 11 has a nice feature to automatically choose an accent color based upon your wallpaper. It works well, but it stops a little short of what it could do.

There are some nice apps like https://github.com/krlvm/AccentColorizer which take it even further.

I'm currently editing a 3rd party .msstyles theme to change what I think are the Highlight * HotTrackingColor, but it would be a lot more convenient to do it in memory & better twice over to pull the automatic accent color.

I think windhawk will be a popular tool among the theming community once they hear about it. Especially if it can remove signature .msstyles files like SecureUxTheme & a couple of other tools.  A lot of theme packs are distributed with stuff like RemoveAddressBar.exe... which is not good practice when a tool like windhawk exists.

**Comment by Nimai-HK** [comment]
Is there any possibility to change the windows start button to follow accent color, using xaml and windhawk?
I don't think it's possible to make it sync to accent color.
